lib/matplotlib/widgets.py

```diff
@@ -2511,27 +2542,31 @@ def onselect(eclick: MouseEvent, erelease: MouseEvent)
         self.spancoords = spancoords
         self._drawtype = drawtype
 
-        self.maxdist = maxdist
+        self.grab_range = grab_range
 
-        if rectprops is None:
-            props = dict(markeredgecolor='r')
+        if props is None:
+            _handle_props = dict(markeredgecolor='black')
         else:
-            props = dict(markeredgecolor=rectprops.get('edgecolor', 'r'))
-        props.update(cbook.normalize_kwargs(marker_props, Line2D._alias_map))
+            _handle_props = dict(
+                markeredgecolor=props.get('edgecolor', 'black')
+                )
+        _handle_props.update(cbook.normalize_kwargs(handle_props,
+                                                    Line2D._alias_map))
```

Comment:
Whole block is 
```
        props = {
            'markeredgecolor': (props or {}).get('edgecolor', 'black'),
            **cbook.normalize_kwargs(handle_props, Line2D._alias_map)}
```

Um, and did you intentionally change the default from 'r'  to 'black'?

Reply:
Yes, this is to match the default `edgecolor`.